src/components/Button.js

```diff
@@ -4,14 +4,15 @@ const Button = styled.button`
   border-radius: 8px;
   border-style: none;
   padding: 20px;
-  width: 300px;
+  margin: 20px;
+  width: 332px;
   background-color: #c3073f;
   color: #1a1a1d;
   font-weight: bold;
   font-size: 1.5rem;
-  bottom: 10px;
   position: fixed;
-  margin: auto;
+  bottom: 0;
+  box-shadow: 0px 10px 13px -7px #000000, 5px 5px 15px 5px rgba(0, 0, 0, 0);
```

Comment:
Just a random hint - adding some opacity to the color `#00000070` can look quite cool, since the shadow gets a bit softer (just in general, didn't see your current shadows in action)

Reply:
i think about it! thx for the hint :)